android/quest/src/main/java/org/smartregister/fhircore/quest/ui/questionnaire/QuestionnaireViewModel.kt

```diff
@@ -306,6 +296,33 @@ constructor(
     }
   }
 
+  private fun launchPostSaveTasks(
+    subject: Resource,
+    questionnaireConfig: QuestionnaireConfig,
+    bundle: Bundle,
+    questionnaire: Questionnaire,
+  ) =
+    CoroutineScope(dispatcherProvider.default()).launch {
+      generateCarePlan(
+        subject = subject,
+        bundle = bundle,
+        questionnaireConfig = questionnaireConfig,
+      )
+
+      executeCql(
+        subject = subject,
+        bundle = bundle,
+        questionnaire = questionnaire,
+        questionnaireConfig = questionnaireConfig,
+      )
+
+      fhirCarePlanGenerator.conditionallyUpdateResourceStatus(
+        questionnaireConfig = questionnaireConfig,
+        subject = subject,
+        bundle = bundle,
+      )
```

Comment:
broken?

Reply:
Moved this part to a separate [branch](https://github.com/opensrp/fhircore/tree/more-perf-logs)